kong/init.lua

```diff
@@ -170,6 +170,7 @@ function Kong.init()
   singletons.loaded_plugins = assert(load_plugins(config, dao))
   singletons.dao = dao
   singletons.configuration = config
+  singletons.router = nil
```

Comment:
This should go in `singletons.lua` instead, where this table is (pre-)allocated for self-documentation purposes